### DISCUSSION TITLE
Create new databases with prisma

### DISCUSSION THREAD

**Discussion by adornerz** [prisma/prisma#11773] [Q&A] [answered]
So, when i declare a data source in the Prisma schema and run `npx prisma migrate dev` for the first time, Prisma will create a database `mydb` on the given MySQL server for me, but is there any way I can programmatically create a new database using Prisma or is my only resport to run the command?

**Comment by nurul3101** [comment, marked as answer]
Hey @adornerz,

Running `npx prisma migrate dev` does create database if it doesn’t already exists.
Another way of programatically creating a database could be by using raw queries.
For example Executing the below function would create a new database.

```javascript
async function main() {
  const result = await prisma.$executeRaw`CREATE DATABASE prismadb`;
  console.log(result);
}
```
You can learn more about Raw Database Access through these [Guides](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#raw-queries-with-relational-databases)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
